modals-notifications/es5/src/js/modal.js

```diff
@@ -0,0 +1,55 @@
+function Modal(config) {
+	Parent.prototype.constructor.apply(this, arguments);
+
+	this.parent = document.querySelector('.modalJs');
```

Comment:
What if there is a SPA, and there is more than one Modal on the page?
I'd recommend you pass the DOM element in the constructor, instead of looking for it from within the component. That would simplify your application